### ISSUE TITLE
Auto-binding audio and subtitles to video

### ISSUE THREAD

**Issue by Omegaplexx** [POMATu/anime-tool#21]
Здравствуйте! Узнал о вашей программе на одном весьма популярном в России торрент-трекере. Я бы хотел предложить опцию автоматической привязки аудиопотоков и субтитров к видео. Функция реализована в модификации mpv от Adventurer_Kun, который можно найти всё на том же торрент-сайте.

Выглядит всё это следующим образом: 
При запуске видео через mpv плеер запоминает каталог выбранного файла и в ищет в подпапках этого каталога файлы форматов m4a, mp3, aac и др. и srt, ass, ssa и т.д. с идентичным с эпизодом названием, и при их нахождении привязывает к видеофайлу. Если дорожек аудио/субтитров найдено несколько, в mpv появляется возможность выбора предпочитаемой дорожки. Всё просто.

Надеюсь, что не откажете в моей просьбе. Благодарю за полезное приложение :)

**Comment by POMATu** [comment]
Здравствуйте, концепт моей программы - ручной, потому что часто я присобачиваю дорожки или субтитры вообще из другой раздачи. Автоматически оно может работать только если названия у аудио/саб файлов такие же как у видеофайла.

Но ваша идея мне понятна, в целом да, можно было бы добавить дополнительную галочку для автопоиска (не отключая ручной концепт со списками), что бы дорожки появлялись автоматом в плеере
Но перед этим придется сделать выбор дорожки по ID (что бы можно было выбирать дорожки по номеру, когда например есть несколько русских дорожек) что я и делаю сейчас (и не только это), но хз когда доделаю тк времени щас мало.

скиньте плз ссылку на проект от Adventurer_Kun (можно на рутрекере скинуть ну или сюда)

**Comment by Omegaplexx** [comment]
Согласен, опциональная возможность автопоиска была бы наилучшим решением. Вот ссылка на мод: https://rutracker.org/forum/viewtopic.php?t=6345652
